src/components/Links.tsx

```
@@ -1,12 +1,12 @@
-            </div>
-            <div>
-                <a href="#" target="_blank">Zaplytic on Twitter</a>
-            </div>
-        </>
-    );
-}
\ No newline at end of file
+  return (
+    <>
+      <div>
+        <a href="https://github.com/zaplytic" target="_blank">Zaplytic on Github</a>
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)